src/PeopleList.tsx

```diff
@@ -0,0 +1,41 @@
+import { Person } from './types/Person';
+
+type Props = {
+  people: Person[];
+  onSelected: (personSlug: Person) => void;
+  setQuery: (value: string) => void;
+};
+
+export const PeopleList: React.FC<Props> = ({
+  people,
+  onSelected,
+  setQuery,
+}) => {
+  return (
+    <div className="dropdown-menu" role="menu">
+      <div className="dropdown-content">
+        {people.length ? people.map((person) => (
+          <div
+            className="dropdown-item"
+            key={person.slug}
+          >
+            <a
+              href={person.slug}
+              className="has-text-link link"
+              onMouseDown={() => {
+                onSelected(person);
+                setQuery(person.name);
+              }}
```

Comment:
Move it to a separate function to improve the code readability and keep your JSX code clean